lib/chef/knife/bootstrap_windows_winrm.rb

```diff
@@ -37,6 +41,17 @@ class BootstrapWindowsWinrm < Bootstrap
       banner "knife bootstrap windows winrm FQDN (options)"
 
       def run
+        if (Chef::Config[:validation_key] && !File.exist?(File.expand_path(Chef::Config[:validation_key])))
+
+          if !negotiate_auth? && !(locate_config_value(:winrm_transport) == 'ssl')
+            ui.error("Validatorless bootstrap only supported with negotiate authentication protocol and ssl/plaintext transport")
```

Comment:
If you weren't using validatorless bootstrap, you'd still be sending the validator.pem over the wire. So this is a protocol problem, not a validatorless bootstrap problem.